readme.rst

```diff
@@ -243,40 +278,48 @@ populates an issue for them with details of the page they were on when they
 
-You can add redirects to make sure existing links and bookmarks continue working when you move files around.
-To do so, specify the old and new paths in the ``redirects`` setting of the ``custom_conf.py`` file.
+You can add redirects to make sure existing links and bookmarks continue working
+when you move files around. To do so, specify the old and new paths in the
+``redirects`` setting of the ``custom_conf.py`` file.
 
 Add custom configuration
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-To add custom configurations for your project, see the ``Additions to default configuration`` and ``Additional configuration`` sections in the ``custom_conf.py`` file.
-These can be used to extend or override the common configuration, or to define additional configuration that is not covered by the common ``conf.py`` file.
+To add custom configurations for your project, see the ``Additions to default
+configuration`` and ``Additional configuration`` sections in the
+``custom_conf.py`` file. These can be used to extend or override the common
+configuration, or to define additional configuration that is not covered by the
+common ``conf.py`` file.
 
 (Optional) Synchronise GitHub issues to Jira
 --------------------------------------------
 
 If you wish to sync issues from your documentation repository on GitHub to your
-Jira board, configure the `GitHub/Jira sync bot <https://github.com/canonical/gh-jira-sync-bot>`_
-by editing the ``.github/workflows/.jira_sync_config.yaml`` file appropriately.
-In addition to updating this file, you must also apply server configuration
-for this feature to work. For more information, see `server configuration details <https://github.com/canonical/gh-jira-sync-bot#server-configuration>`_
-for the GitHub/Jira sync bot.
+Jira board, configure the `GitHub/Jira sync bot
+<https://github.com/canonical/gh-jira-sync-bot>`_ by editing the
+``.github/workflows/.jira_sync_config.yaml`` file appropriately. In addition to
+updating this file, you must also apply server configuration for this feature to
+work. For more information, see `server configuration details
+<https://github.com/canonical/gh-jira-sync-bot#server-configuration>`_ for the
+GitHub/Jira sync bot.
 
 The ``.jira_sync_config.yaml`` file that is included in the starter pack
-contains configuration for syncing issues from the starter pack repository to 
-its documentation Jira board.
-Therefore, it does not work out of the box for other repositories in GitHub, 
-and you must update it if you want to use the synchronisation feature.
+contains configuration for syncing issues from the starter pack repository to
+its documentation Jira board. Therefore, it does not work out of the box for
+other repositories in GitHub, and you must update it if you want to use the
+synchronisation feature.
 
 Change log
 ----------
 
-See the `change log <https://github.com/canonical/sphinx-docs-starter-pack/wiki/Change-log>`_ for a list of relevant changes to the starter pack.
+See the `change log
```

Comment:
I do not know from the top of my head whether this works, or whether you need to have the link on a single line, which I noted I would not mind if that is the case.

We have a couple of those cases in that file. If this works, ok, go forward, if not you could use the syntax where you only have the link text in the paragraph and the link syntax only used after the paragraph, see third example in https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages